source/edge-server/configure.txt

```diff
@@ -10,4 +10,362 @@ Configure Edge Server
    :depth: 2
    :class: singlecol
 
-This is a placeholder page.
+Edge Server is a "local" server that sits between your client devices and 
+the MongoDB Atlas App. For most use cases, you'll provision hardware 
```

Comment:
nit I would replace "Atlas App" with "Atlas App Services" here and in the next paragraph